pages/k8s/1.27/components.md

```diff
@@ -99,7 +99,7 @@ to extend the capabilities of your Kubernetes cluster.
 |charm | summary | source | docs | bugs |
 |-|-|-|-|-|
 | [aws-iam](https://charmhub.io/aws-iam) |  | [source](https://github.com/charmed-kubernetes/charm-aws-iam.git) | [docs](https://charmhub.io/aws-iam/docs)  | [bugs](https://bugs.launchpad.net/charm-aws-iam) |
-| [ceph-csi](https://charmhub.io/ceph-csi) | Deploys Container Storage Interface (CSI) plugin that enables Charmed Kubernetes to use ceph as a storage backend. | [source](https://github.com/charmed-kubernetes/ceph-csi-operator.git) | [docs](https://charmhub.io/ceph-csi/) | [bugs](https://bugs.launchpad.net/charm-ceph-csi) |
+| [ceph-csi](https://charmhub.io/ceph-csi) | Deploys Container Storage Interface (CSI) plugin that enables Charmed Kubernetes to use ceph as a storage backend. | [source](https://github.com/charmed-kubernetes/ceph-csi-operator.git) | [docs](https://charmhub.io/ceph-csi/docs)  | [bugs](https://bugs.launchpad.net/charm-ceph-csi) |
```

Comment:
this isn't right. I mean, it works but it just gets redirected back to charmhub.io/ceph-csi

Reply:
yeah, but everything else does it :)

this comes from our [charm-support-matrix.inc](https://github.com/charmed-kubernetes/jenkins/blob/main/jobs/includes/charm-support-matrix.inc#L48) and is consistent with the other charm `docs:` keys.